src/components/experiment-tracking/parallel-coordinates/parallel-coordinates.js

```diff
@@ -12,6 +15,9 @@ const paddingLr = 80;
 const axisGapBuffer = 3;
 const selectedMarkerRotate = [45, 0, 0];
 
+const sideBarWidth = 540;
```

Comment:
Where is this value derived from? I think it's actually `480`, and we have a `const` for this in the config file [here](https://github.com/kedro-org/kedro-viz/blob/main/src/config.js#L16) that you can use instead of hard coding it.

Reply:
ah yeah i was looking for it, thank you